delphi/translators/for2py/pyTranslate.py

```diff
@@ -703,7 +706,28 @@ def printFuncReturn(self, node, printState: PrintState):
             if node.get("name") is not None:
                 val = self.nameMapper[node["name"]] + "[0]"
             else:
-                val = node["value"]
+                if "value" in node:
+                    val = node["value"]
+                else:
+                    assert (
+                        "left" in node and
+                        "operator" in node and
+                        "right" in node
+                    ), f"Something is missing. Detail of node content: {node}"
+
+                    left = ""
```

Comment:
This assignment seems unnecessary, since the variable is overwritten in the if statement.